crypto/include/fdoCrypto.h

```diff
@@ -14,16 +14,14 @@
 #include "base64.h"
 #include "fdoCryptoCtx.h"
 
-#ifdef KEX_ECDH384_ENABLED
-#define SEK_KEY_SIZE 32
-#define SVK_KEY_SIZE 64
+#if defined KEX_ECDH384_ENABLED
+#define SEK_KEY_SIZE 32 /* KEX_ECDH384_ENABLED */
 #else
 #define SEK_KEY_SIZE 16
-#define SVK_KEY_SIZE 32
-#endif /* KEX_ECDH384_ENABLED */
+#endif
 
-/* Cipher suite "cs" for msg 40 in TO2 */
-#ifdef AES_256_BIT
+/* Cipher suite "cs" for msg 60 in TO2 */
```

Comment:
msg 60 => msg60 .. easier to get through grep commands later - most of the usage is in that format.